src/stats.py

```diff
@@ -30,14 +30,14 @@ def __init__(self, chat_json: Union[str, Path]):
         """
         # load chat data
         logger.info(f"Loading chat data from {chat_json}")
-        with open(chat_json) as f:
+        with open(chat_json, encoding='UTF8') as f:
```

Comment:
Hello Ali and thanks for the PR.
Encoding of JSON files extracted from Telegram is UTF-8 and It is a good idea to identify the encoding of files when reading them. Thanks for this suggestion.